packages/odyssey-react-mui/src/OdysseyI18n.ts

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+
+import { translation as en } from "./properties/ts/odyssey-react-mui";
+import { translation as fr } from "./properties/ts/odyssey-react-mui_fr";
```

Comment:
Are supported languages will have to be imported as such once translated

Reply:
Ah, so this is a manual process? Is there a way to look at the `translations/` dir and automate that process as part of the TS file creation? I'm sure we could write in a simple Node.js script to do it.